subsys/bluetooth/controller/ll_sw/lll_scan.h

```diff
@@ -86,3 +86,4 @@ int lll_scan_reset(void);
 void lll_scan_prepare(void *param);
 
 extern uint8_t ull_scan_lll_handle_get(struct lll_scan *lll);
+extern struct lll_scan *ull_scan_lll_is_valid_get(struct lll_scan *lll);
```

Comment:
```suggestion
extern struct lll_scan *ull_scan_lll_get_valid(struct lll_scan *lll);
```

Reply:
It is common signature in zephyr atleast when returning boolean to use `something_is_valid()`, hence Controller uses `something_is_valid_get()` to return a context if valid within its memory pool.

Reply:
> It is common signature in zephyr atleast when returning boolean to use something_is_valid()

Indeed it is, but this is also one my main peeves with this name: using `is_valid` in a function that does not return a boolean. 

I'd like a 3rd input on this as well (ping @ppryga @kruithofa ), although I will not block this PR due to this :)

Reply:
I got used to the `is_valid_get` but I agree with @Thalley. If one reads the name first time, she may be confused: is this a bool value that is returned or if `is valid` then `get` reference.

Reply:
IMO 'ull_scan_lll_get_valid' is a better name, since this function is not returning a boolean.
However that will break consistency with already existing functions that follow the scheme 'something_is_valid', in particulare we already have an 'ull_scan_is_valid_get'.

It is probably best to create an issue/enhancement proposing the name change, and then do the change for all functions in one go, so that we remain consistent.

Reply:
Zephyr is a mix of `something_get()` and `get_something()`, hence will be a API breaker enhancement needing some real work to get consistency, else the naming preference will always come up again in reviews.

Reply:
> Zephyr is a mix of `something_get()` and `get_something()`, hence will be a API breaker enhancement needing some real work to get consistency, else the naming preference will always come up again in reviews.

From https://github.com/zephyrproject-rtos/zephyr/pull/30892 it seems that the majority points towards `get_something()` for `get`ters. 

I agree with @kruithofa that it's probably better to do the consisting naming refactor in another PR.

Reply:
#30892 has not concluded. The last agreement being from @nashif  “noun-verb” word ordering is recommended, example `k_work_delayable_remaining_get()` given here: https://github.com/zephyrproject-rtos/zephyr/pull/30892/files#r556478744

And the approved API is upstream: https://github.com/zephyrproject-rtos/zephyr/blob/2f5976d674fff5dd437074a2a92ae890c674e594/include/kernel.h#L3318

`ull_scan_lll_is_valid_get`  is descriptive of the purpose, is to check for valid context and then return the context, not a boolean.
`ull_scan_lll_valid_get` or `ull_scan_lll_get_valid` could mean `valid` being the noun/subject.